### ISSUE TITLE
http://download.eclipse.org/jdtls/snapshots/jdt-language-server-0.68.0-202101202016.tar.gz doesn't exist and breaks install on Mac

### ISSUE THREAD

**Issue by pwolter** [macvim-dev/macvim#1223]
### Steps to reproduce

Follow installation instructions for Mac OS here: https://ycm-core.github.io/YouCompleteMe/#installation

1. brew install cmake macvim python
2. brew install mono go nodejs
3. cd ~/.vim/bundle/YouCompleteMe
python3 install.py --all 

### Expected behaviour

The install breaks at some point:

Rust is installed now. Great!

To get started you need Cargo's bin directory
(/var/folders/m0/6fmggpxj1vzcp5wjftfx__7r0000gp/T/rust_install_lng2fptu/bin) in
your PATH
environment variable. This has not been done automatically.

To configure your current shell, run:
source
/var/folders/m0/6fmggpxj1vzcp5wjftfx__7r0000gp/T/rust_install_lng2fptu/env
info: syncing channel updates for 'nightly-2021-07-29-x86_64-apple-darwin'
info: latest update on 2021-07-29, rust version 1.56.0-nightly (b70888601 2021-07-28)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
 17.0 MiB /  17.0 MiB (100 %)  15.2 MiB/s in  1s ETA:  0s
info: downloading component 'rust-std'
 21.0 MiB /  21.0 MiB (100 %)  19.6 MiB/s in  1s ETA:  0s
info: downloading component 'rustc'
 75.8 MiB /  75.8 MiB (100 %)  23.7 MiB/s in  3s ETA:  0s
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
 17.0 MiB /  17.0 MiB (100 %)   5.7 MiB/s in  2s ETA:  0s
info: installing component 'rust-std'
 21.0 MiB /  21.0 MiB (100 %)  15.1 MiB/s in  1s ETA:  0s
info: installing component 'rustc'
 75.8 MiB /  75.8 MiB (100 %)  16.0 MiB/s in  4s ETA:  0s
info: installing component 'rustfmt'

  nightly-2021-07-29-x86_64-apple-darwin installed - rustc 1.56.0-nightly (b70888601 2021-07-28)

info: default toolchain set to 'nightly-2021-07-29-x86_64-apple-darwin'
info: checking for self-updates
info: downloading component 'rust-src'
info: installing component 'rust-src'
info: downloading component 'rust-analyzer-preview'
info: installing component 'rust-analyzer-preview'
info: component 'rustfmt' for target 'x86_64-apple-darwin' is up to date
info: component 'clippy' for target 'x86_64-apple-darwin' is up to date
Downloading jdt.ls from http://download.eclipse.org/jdtls/snapshots/jdt-language-server-0.68.0-202101202016.tar.gz...
Traceback (most recent call last):
  File "/Users/pwolter/.vim/bundle/YouCompleteMe/third_party/ycmd/build.py", line 1185, in <module>
    Main()
  File "/Users/pwolter/.vim/bundle/YouCompleteMe/third_party/ycmd/build.py", line 1177, in Main
    EnableJavaCompleter( args )
  File "/Users/pwolter/.vim/bundle/YouCompleteMe/third_party/ycmd/build.py", line 992, in EnableJavaCompleter
    DownloadFileTo( url, file_name )
  File "/Users/pwolter/.vim/bundle/YouCompleteMe/third_party/ycmd/build.py", line 134, in DownloadFileTo
    with urllib.request.urlopen( download_url ) as response:
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/opt/anaconda3/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

### Version of Vim

VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Apr 01 2021 08:32:44)

### Environment

MacOS 10.15.7
Term: xterm-256color
zsh shell

**Comment by eirnym** [comment]
@pwolter Thank you for an issue

TL;DR: Please, file this issue to YCM directly

Installation of YouCompleteMe doesn't touch Vim's interfaces, so I don't see any relation with Vim or MacVim, but YouCompleteMe, even it contains the vim scripts.
The error says, YCM setup tool tries to download some version of JDT language server which doesn't exists anymore.
